exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,15 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
```

Comment:
Alright this is good. Remember,the dollar sign is the placeholder for the variables we create.